server.js

```diff
@@ -2,18 +2,27 @@ import express from "express";
 import morganLogger from "morgan";
 import querystring from "querystring";
 import axios from "axios";
+import envalid from 'envalid'
+const { str, num, url } = envalid;
 import cookieParser from "cookie-parser";
 import { generateRandomString } from "./utils";
 
+const CleanEnv = envalid.cleanEnv(process.env, {
```

Comment:
Use camelcase. There will be a linter error for this soon
```suggestion
const cleanEnv = envalid.cleanEnv(process.env, {
```